src/TodoList.jsx

```diff
@@ -0,0 +1,25 @@
+function TodoList() {
+  const todoList = [
```

Comment:
So you actually want your `todoList` outside of your `TodoList` component. You can have it above, outside of your component.